helm/dagster/values.yaml

```diff
@@ -916,18 +916,20 @@ dagsterDaemon:
   heartbeatTolerance: 300
 
   runCoordinator:
-    enabled: false
+    enabled: true
 
     # Type can be one of [
     #   QueuedRunCoordinator,
     #   CustomRunCoordinator,
     #  ]
     type: QueuedRunCoordinator
     config:
+      # https://docs.dagster.io/deployment/run-coordinator#limiting-run-concurrency
       queuedRunCoordinator:
-        maxConcurrentRuns: ~
+        maxConcurrentRuns: 10
```

Comment:
These defaults were getting populated in python- easier to let users see them here

Reply:
not sure if these defaults will change in the python side. Might be better to leave these empty, and leave a link to the API docs that shows the defaults? I believe #6535 should show this now on the API docs

Reply:
I believe https://docs.dagster.io/_apidocs/internals#dagster.core.run_coordinator.QueuedRunCoordinator should have this info after the docs site is updated

Reply:
Looks like it has the signature. I think we could move the non-list defaults there potentially?  `(max_concurrent_runs=None, tag_concurrency_limits=None, dequeue_interval_seconds=None, inst_data=None)`

But alternatively we could just add a comment in the code to change the helm defaults if we ever change these